src/components/Socials/styles.scss

```
@@ -1,3 +1,64 @@
+// Mobile portrait
 .wrapper {
+	display: flex;
+	justify-content: center;
+	width: 100%;
 	font-size: 1em; //nothing comment to make sure class appears
+	
+		li {
+			width: 1em;
+			height: 1em;
+			list-style: none;
```

```suggestion
			list-style: none;
			&+.item {
				margin-left: 4em;
			}
```
Implementing the rule you'd established in the `Navigation` styles here instead of up there.